test/integration/controller/admissionchecks/provisioning/provisioning_test.go

```diff
@@ -592,15 +597,32 @@ var _ = ginkgo.Describe("Provisioning", ginkgo.Ordered, ginkgo.ContinueOnFailure
 			})
 
 			ginkgo.By("Checking the admission check is ready", func() {
+				updatedWl := &kueue.Workload{}
```

Comment:
Should we add additional step "Check the workload is admitted"? If the workload isn't admitted, we will invoke 
`syncOwnedProvisionRequest` which may recreate the PR, and thus evict the workload. See https://github.com/kubernetes-sigs/kueue/blob/main/pkg/controller/admissionchecks/provisioning/controller.go#L138-L151

Reply:
SGTM

Reply:
Done.